src/Packetery/Core/Api/Soap/Client.php

```diff
@@ -9,8 +9,6 @@
 
 namespace Packetery\Core\Api\Soap;
 
-use Packetery\Core\Api\Soap\Request;
-use Packetery\Core\Api\Soap\Response;
```

Comment:
why this wasn't covered by phcps?

Reply:
Very good idea, is it possible to include in this PR, or should we add it together with other missing useful rules?

Reply:
please in the new MR.

Reply:
Ticket is on the way.